eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -51,9 +51,6 @@ trait AbstractCommitments {
   /** Returns all HTLCs in the current remote commitment. */
   def htlcsRemoteCommit: Set[DirectedHtlc]
 
-  /** Returns all HTLCs in the current local commitment. */
-  def htlcsLocalCommit: Set[DirectedHtlc]
```

Comment:
I'm not sure it makes sense to introduce this yet because so far it's not used anywhere except Commitments object itself.